action-processor/create-action.js

```diff
@@ -444,6 +448,26 @@ async function createMarketingAuthorisationRenewalJob (featureName, action){
     localStorage.setItem(featureName, 'maRenewalJobList', maRenewalJobList)
 }
 
+async function createMarketingAuthorisationNewJob (featureName, action){
+    log.debug(`${SERVICE_NAME}::createMarketingAuthorisationNewJob`)
+    let data = action.data
+    log.info(`${SERVICE_NAME}::createMarketingAuthorisationNewJob::${action.label}::creating marketing authorisation new job from ${JSON.stringify(data)}`)
+    let responseData = await jobService.createJob(constants.JOB_TYPE_MARKETING_AUTHORISATION_NEW, featureName)
+    let jobId = responseData.Id
+    responseData = await jobService.updateJob(jobId, data)
+    log.info(`${SERVICE_NAME}::createMarketingAuthorisationNewJob::${action.label}::updated:${JSON.stringify(responseData)}`)
+    var savedAction = localStorage.getItem(featureName, action.label)
+    savedAction.response = responseData
+    log.debug(`${SERVICE_NAME}::createMarketingAuthorisationNewJob, saved action ${JSON.stringify(savedAction)}`)
+    localStorage.setItem(featureName, action.label, savedAction)
+    var maNewJobList = localStorage.getItem(featureName, 'maNewJobList')
+    if (!maNewJobList) {
+        maNewJobList = []
+    }
+    maNewJobList.push(responseData.Id)
```

Comment:
you could change `responseData.Id` to `jobId`

Reply:
changed to responseData.Identifier which the other use. Ty for pointing out